xrpl4j-core/src/main/java/org/xrpl/xrpl4j/model/transactions/metadata/MetaLedgerEntryType.java

```diff
@@ -39,6 +52,49 @@ static MetaLedgerEntryType of(String value) {
       .build();
   }
 
+  /**
+   * Get the {@link MetaLedgerObject} concrete type associated with this {@link MetaLedgerEntryType}.
+   *
+   * @return A {@link Class} of {@link MetaLedgerObject}.
+   */
+  @Value.Auxiliary
```

Comment:
This could technically be `@Derived`. `@Auxiliary` is basically `@Derived` but it excludes that field from `equals`, `hashCode` and `toString` methods. I don't have a strong opinion about whether to use `@Derived` or `@Auxiliary` here, but this field is _definitely_ supposed to be JSON ignored.

Reply:
Changed to `@Derived` and added `@JsonIgnore`